config/tests/samples/create/samples.go

```diff
@@ -133,6 +133,7 @@ func RunCreateDeleteTest(t *Harness, opt CreateDeleteTestOptions) {
 				t.Fatalf("error creating resource: %v", err)
 			}
 		} else {
+			t.Log("using server-side apply to create object")
```

Comment:
do we need these log lines? if so, can we add the resource as the identifier?

Reply:
The log can be very helpful to identify whether SSA is used. Added more info as the identifier.